src/Views/SuccessView.vala

```diff
@@ -62,12 +102,6 @@ public class SuccessView : AbstractInstallerView {
         action_area.add (shutdown_button);
         action_area.add (restart_button);
 
-        Timeout.add_seconds (RESTART_TIMEOUT, () => {
```

Comment:
I _think_ this should remain upstream, as we do want the countdown for automatically restarting. It was removed downstream for Pop!